cleaner/scheduled_tasks/db/install.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<XMLDB PATH="local/datacleaner/cleaner/scheduled_tasks/db" VERSION="2019030400" COMMENT="XMLDB file for Moodle local/datacleaner/cleaner/scheduled_tasks"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="../../../../../lib/xmldb/xmldb.xsd"
+>
+    <TABLES>
+        <TABLE NAME="cleaner_scheduled_tasks" COMMENT="Scheduled tasks config - whether to disable the task in a clean">
+            <FIELDS>
+                <FIELD NAME="id" TYPE="int" LENGTH="10" NOTNULL="true" SEQUENCE="true"/>
+                <FIELD NAME="task_scheduled_id" TYPE="int" LENGTH="10" NOTNULL="true" SEQUENCE="false"/>
```

Comment:
I can't actually find it anywhere in the coding style guide, but table fields elsewhere in Moodle don't use underscores, so let's stick with this convention.

Reply:
docs are here:

https://docs.moodle.org/dev/XMLDB_defining_an_XML_structure#Conventions